src/aeonlib/ocs/blanco/instruments.py

```diff
@@ -1,17 +1,32 @@
 # pyright:  reportUnannotatedClassAttribute=false
 # This file is generated automatically and should not be edited by hand.
 
-from typing import Any, Annotated, Literal
+from typing import Any, Annotated, Literal, Optional
```

Comment:
I can see this being a pain because this is generated code, and if there doesn't happen to be any optional attributes then this warning doesn't make sense.

Luckily, using Optional is discouraged. The correct way to annotate optional types now is to use `Type | None` like `defocus: float | None = None`

Reply:
That's good to know, I was figuring out pydantic and jinja2 as i went...